src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/ISwitchCarrier.java

```diff
@@ -41,4 +42,6 @@
     void sendSwitchSynchronizeRequest(String key, SwitchId switchId);
 
     void sendAffectedFlowRerouteRequest(SwitchId switchId);
+
+    void sendUpdateSingleSwitchFlowsRequest(SwitchId switchId, SwitchStatus status);
```

Comment:
Bad name. Why if mention "single-switch"? It must be something like `sendSwitchStatusUpdateNotification`, the receiver can treat this notification in any required way.

Reply:
Agree, although there's a precedent with sending "flow reroute" request instead of "ISL status update" notification - just check the method above.